### PR TITLE
Add information about relations to extended wikitext reader

### PR DESCRIPTION
**Pull request by nelson-liu**

### CODE REVIEW

File: kglm/data/dataset_readers/enhanced_wikitext.py [hunk 1]
```diff
@@ -213,6 +223,15 @@ def text_to_instance(self, data: Dict[str, Any]) -> Instance:  # pylint: disable
...
                 dtype=np.int64)
+            fields['parent_ids'] = TextField(
+                [Token(x) for x in parent_ids],
+                token_indexers=self._entity_indexers)
+            fields['relations'] = TextField(
+                [Token(x) for x in relations],
+                token_indexers=self._entity_indexers)
+            fields['has_parent_mask'] = SequentialArrayField(
+                np.array(has_parent_mask, dtype=np.int64),
+                dtype=np.int64)
```
Comment (rloganiv) [inline kglm/data/dataset_readers/enhanced_wikitext.py]: `dtype` should be `uint8`

File: kglm/data/dataset_readers/enhanced_wikitext.py [hunk 2]
```diff
@@ -193,6 +197,12 @@ def text_to_instance(self, data: Dict[str, Any]) -> Instance:  # pylint: disable
                     shortlist.append(entity_id)
                 shortlist_ind = reverse_shortlist[entity_id]
 
+                # Collect the relations and their parents
+                relations.extend(annotation["relation"])
+                parent_ids.extend(annotation["parent_id"])
```
Comment (rloganiv) [inline kglm/data/dataset_readers/enhanced_wikitext.py]: With this approach how can you determine which annotation each relation and parent id come from?

Comment (nelson-liu) [reply kglm/data/dataset_readers/enhanced_wikitext.py]: right, this was why i was asking about the linking the (entity, relation, parentid) triples last night

**Comment by nelson-liu** [comment]
subsumed by #25

**Pull request closed without merging**